### PR TITLE
CCLens3d can be concave

### PR DESCRIPTION
**Pull request by billy1380**

### CODE REVIEW

File: cocos2dx/actions/CCActionGrid3D.h [hunk 1]
```diff
@@ -96,7 +96,10 @@ class CC_DLL CCLens3D : public CCGrid3DAction
     inline float getLensEffect(void) { return m_fLensEffect; }
     /** Set lens center position */
     inline void setLensEffect(float fLensEffect) { m_fLensEffect = fLensEffect; }
-    
+    /** Set whether lens is concave */
+    inline void setConcave(float fLensEffect) { m_bConcave = fLensEffect; }
```
Comment (minggo) [inline cocos2dx/actions/CCActionGrid3D.h:100]: Why the type of parameter is float?
Isn't it boolean?

**Comment by minggo** [comment]
What changes were done to cocos2dx/proj.ios/cocos2dx.xcodeproj/project.pbxproj?

File: cocos2dx/actions/CCActionGrid3D.h (hunk 1 above)
Comment (billy1380) [inline cocos2dx/actions/CCActionGrid3D.h:100]: It should be:

 inline void setConcave(bool bConcave) { m_bConcave = bConcave; }

Very sorry... copy paste mistake

**Comment by billy1380** [comment]
My default XCode settings are 2 spaces for tab, so I changed the project setting so that it is 4 spaces per tab to match the rest of the code files...

**Comment by billy1380** [comment]
minggo, just reject it... I will send you another pull request and try not to make so many mistakes

**Pull request closed without merging**